src/resolve/adapters/digitalExperienceSourceAdapter.ts

```diff
@@ -144,7 +193,47 @@ export class DigitalExperienceSourceAdapter extends BundleSourceAdapter {
     }
   }
 
+  private populateWebAppBundle(trigger: string, component?: SourceComponent): SourceComponent {
+    if (component) {
+      return component;
+    }
+
+    const pathParts = trigger.split(sep);
+    const digitalExperiencesIndex = pathParts.indexOf('digitalExperiences');
+
+    // Extract bundle name: web_app/WebApp3
+    const baseType = pathParts[digitalExperiencesIndex + 1];
+    const spaceApiName = pathParts[digitalExperiencesIndex + 2];
+    const bundleName = `${baseType}/${spaceApiName}`;
```

Comment:
I know server responses use `/` but if this is going to a filesystem, you need to use `join`